packages/shared/src/components/plus/PlusMobile.tsx

```diff
@@ -36,6 +39,7 @@ export const PlusMobile = (): ReactElement => {
         selectedOption={selectedOption}
         onChange={selectionChange}
         onContinue={onContinue}
+        shouldShowPlusHeader={shouldShowPlusHeader}
```

Comment:
Not blocking for this PR but this kind of thing could be passed to child components from some kind of `PlusPageContext`, similar how we do it for search panel, or feed settings. It will avoid props drilling and make adding new stuff easier.

Reply:
Totally agree, I wanted to do that too, it just felt like the header condition does not fit the narrative of the context. Maybe we can do it during the clean-up if this wins. Wdyt?

Reply:
Of course 🤝

Reply:
I will do it in a separate PR to not block this and carefully test it 🙏